Waste.API/Data/WasteContext.cs

```diff
@@ -14,5 +18,18 @@ public WasteContext(DbContextOptions options) : base(options)
         public DbSet<WasteSubType> WasteSubType { get; set; }
 
         public DbSet<WasteJourney> WasteJourney { get; set; }
+
+        protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
+        {
+            if (!optionsBuilder.IsConfigured)
+            {
+                IConfigurationRoot configuration = new ConfigurationBuilder()
+                   .SetBasePath(Directory.GetCurrentDirectory())
+                   .AddJsonFile("appsettings.json")
+                   .Build();
+                var connectionString = configuration.GetConnectionString("WasteConnectionString");
+                optionsBuilder.UseSqlServer(connectionString);
+            }
+        }
```

Comment:
WasteContext is already configured in Program.cs file and therefore you don't need this OnConfiguring method.

Reply:
Done.